docs/make_conf.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+
+import argparse
+from string import Template
+
+parser = argparse.ArgumentParser(description='Make configuration file for sphinx docs')
+parser.add_argument('--project')
+parser.add_argument('--infile', default='conf_template.py')
+parser.add_argument('--outfile', default='conf.py')
+
+
+def main(infile, outfile, project):
+    with open(infile, "r") as inf:
+        contents = inf.read()
+    template = Template(contents)
+
+    with open(outfile, "w") as outf:
+        outf.write("##############################################################\n")
+        outf.write("# WARNING! This file was generated by make_conf.py.\n")
+        outf.write("# Do not update this file in-place.\n")
+        outf.write("# Use make_conf.py to create a new file.\n")
```

Comment:
You could use `__file__` here instead of `make_conf.py` to make this robust against changes of the file name.